examples_tests/tests.py

```diff
@@ -178,6 +178,9 @@ class TestSnapcraftExamples(testscenarios.TestWithScenarios):
             'dir': 'ros',
             'name': 'ros-example',
             'version': '1.0',
+            'external_tests_commands': [
+                ("sed -n '/env/p;1q' snap/usr/bin/rosversion",
+                 b'#!/usr/bin/env python\n',)],
```

Comment:
how clever this guy who invented the external test commands ;)
I would explain in a comment what that sed statement is doing there.